### PR TITLE
[PAGINATION] Add Pagination to Categories

### PR DESCRIPTION
**Pull request by kamp3r**
As administrator
I WANT to view the results of Paginated Categories
TO increase response speed

Criteria of acceptance:
When the request to list is made, the results should be paginated by 10. In the response, the results and the urls for the previous and next page (if applicable) should be shown. The current page will be obtained from the query parameter ?page

*The adjustments of this PR were discussed on daily

### EVIDENCE

ot13

ot14

### CODE REVIEW

File: routes/categories.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const { authUserAdmin } = require('../middlewares/authUserAdmin')
 
 const router = Router()
 
-router.get('/public', authUserAdmin, get)
+router.get('/', authUserAdmin, get)
```
Comment (ghnoob) [inline routes/categories.js:15]: no tendria que ser `/public` como decia en la card vieja el endpoint?

**Review by bgalacho — APPROVED** [review]

File: routes/categories.js (hunk 1 above)
Comment (kamp3r) [reply routes/categories.js:15]: En realidad la card vieja no pedia eso , por eso aproveche y lo corregi

imagen

**Review by ghnoob — APPROVED** [review]
Todo perfecto

**Review by tomasignacioc — APPROVED** [review]
impresionante

**Review by Samuel0023 — APPROVED** [review]

**Review by gabodmc — APPROVED** [review]
Esta bien. Solo un comentario:
La paginacion se podia meter adentro del servicio que ya tenian sin cambiarle el nombre

**Pull request merged**